src/main/java/Duke.java

```diff
@@ -1,10 +1,72 @@
-public class Duke {
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.DukeException;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.stage.Stage;
+
+/**
+ * CS2103T AY 19/20 S2
+ * Individual Project
+ * @author Yap Dian Hao
+ * @since 2020-01-30
+ * <p>
+ * Duke is a personal chatbot assistant to help you organize your tasks. Especially helpful
+ * for people who love to procrastinate. Or procrastinate to procrastinate.
+ * </p>
+ */
+
+public class Duke extends Application {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructor of the Duke chatbot assistant.
+     * Duke is able to perform CRUD operations (Create, Read, Update, Destroy) to manage tasks.
+     * @param filePath A text file which functions as a database to store all tasks.
+     */
+
```

Comment:
Hi, may I ask that is the default constructor auto-generated? As I think the java doc comment is for the second constructor, and sometimes auto-generating code will have this issue, hehe.